v19.2/cockroachcloud-connect-to-your-cluster.md

```diff
@@ -61,11 +61,11 @@ Use the Console to authorize networks:
 
     The **Add User** modal displays.
 
-    <img src="{{ 'images/v19.2/cockroachcloud/add-user-modal.png' | relative_url }}" alt="Add user" style="border:1px solid #eee;max-width:50%" />
-
-3. Enter a **Username**.
-4. Enter and confirm the **Password**.
-5. Click **Create**.
+3. Enter a **Username** and **Password**.
+    {{site.data.alerts.callout_info}}
+    Password should be at least 12 characters long. 
+    {{site.data.alerts.end}}
+4. Click **Create**.
 
     Currently, all new users are created with Admin privileges. For more information and to change the default settings, see [Granting privileges](cockroachcloud-authorization.html#granting-privileges) and [Using roles](cockroachcloud-authorization.html#using-roles).
```

Comment:
Just noticed while scrolling through. 

- Should be `admin` and not Admin
- Based on my current (shaky) understanding, `admins` have the `ALL` privilege. There are no "admin" privileges. I know that "admin privileges" is a common phrase, but since we emphasize that `admin` is a role, perhaps this should be phrased: "Currently, all new users belong to the `admin` role by default."